backend/src/types.ts

```diff
@@ -25,15 +25,16 @@ export type DashboardConfig = K8sResourceCommon & {
         enabled: boolean;
       };
     };
-    notebookControllerState?: [
-      {
-        user: string;
-        lastSelectedImage: string;
-        lastSelectedSize: string;
-        environmentVariables: EnvironmentVariable[];
-        secrets: string;
```

Comment:
Could you refrain from removing anything from the block? This PR should just change the location.